pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -61,15 +80,13 @@ public void validate(TokenCredentials credentials) {
 
         try {
             // Parse the token
-            JWT jwt = JWTParser.parse(token);
+            final JWT jwt = JWTParser.parse(token);
 
             if (jwt instanceof SignedJWT) {
                 signedJWT = (SignedJWT) jwt;
             } else if (jwt instanceof EncryptedJWT) {
-                JWEObject jweObject = (JWEObject) jwt;
-
-                // Decrypt with shared key
-                jweObject.decrypt(new DirectDecrypter(this.secret.getBytes("UTF-8")));
+                final JWEObject jweObject = (JWEObject) jwt;
```

Comment:
There should be a `CommonHelper.assertNotBlank("encryptionSecret", encryptionSecret);` here.